.github/workflows/dev.yml

```diff
@@ -0,0 +1,15 @@
+name: Build and push the dev image
+
+on:
+  push:
+    branches: [ feature/dockerfile ]
```

Comment:
@jaensen is this still correct as is?

Reply:
Hi! These are the workflows for our docker image build. They'll most likely won't work outside of our GitHub organization :/
You can either ignore/remove it or we can work together with @almereyda to setup the stuff in a way that will also work from your repository.